charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,63 @@
+
+# Metadata Schema
+
+## Description
+
+The HCA Metadata schema committee creates and maintains the Human Cell Atlas metadata schema in JSON format. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure the FAIRness* of the HCA data [The FAIR Guiding Principles for scientific data management and stewardship](https://www.nature.com/articles/sdata201618). 
+
+## Objective
+
+1. Ensure that HCA data are findable and the schemas are computationally queryable.
+2. Ensure that HCA metadata supports reproducibility and reusability.
+3. Ensure that HCA metadata has a consistent representation and is computationally queryable.
+4. Define JSON schema which enable objectives 1 and 2.
+5. Specify the entity structure for HCA metadata schema.
+6. Define style guides and release process.
+7. Ensure the defined metadata schema provides a positive user experience for those who read and write them.
```

Comment:
nit: These are not full sentences. I would omit the period at the end of each.

Reply:
Thanks, updates made